app/controllers/conversations.js

```diff
@@ -216,24 +217,86 @@ class ConversationsController extends BaseController {
           validateParticipantsLimit,
         ]);
 
-        const participants = [];
-        for (let i = 0; i < addUsers.length; i++) {
+        const participants = await Promise.all(
+          addUsers.map(async (uId) => {
+            if (
+              !(await ConversationParticipant.findOne({
+                user_id: uId,
+                conversation_id: conversationId,
+              }))
+            ) {
+              return uId;
+            }
+          })
+        );
```

Comment:
do we need to add `.filter(p => !!p)` to filter `undefined` values when user is already in chat?